[Docs] Metrics configured in CM (#3948)
strimzi/strimzi-kafka-operator: documentation/assemblies/metrics/assembly_metrics-cruise-control.adoc

== Configuring Cruise Control

> You can enable the Cruise Control metrics in the `Kafka` resource using the `cruiseControl.metricsConfig` property that contains a reference to the ConfigMap containing the JMX exporter configuration about the metrics to expose.

Reviewer: ```suggestion
Enable Cruise Control metrics using the `cruiseControl.metricsConfig` property in the `Kafka` resource to provide a reference to a ConfigMap that contains JMX exporter configuration about the metrics to expose.
```

Reviewer: Would an example fragment from a ConfigMap help here?

Author: I guess this comment applies to all resources, not just cruise control. Is that correct?
